mirascope/beta/openai/realtime/realtime.py

```diff
@@ -95,7 +136,7 @@ def __init__(
             "tool_choice": tool_choice,
             "temperature": temperature,
             "max_response_output_tokens": max_response_output_tokens,
-            "tools": tools,
+            "tools": self._tool_schemas.values() if self._tool_schemas else NotSet,
```

Comment:
again why aren't we just using our existing tool schema utilities from `OpenAITool`??

Reply:
https://platform.openai.com/docs/api-reference/realtime-client-events/session/update
<img width="428" alt="image" src="https://github.com/user-attachments/assets/b28ae435-08e6-4e31-ba45-260e8dc61474">

↑we need the schema for Realtime API. But, `OpenAITool.tool_schema()` retruns `ChatCompletionToolParam`
which is diffrent type.
<img width="978" alt="image" src="https://github.com/user-attachments/assets/fd0126f9-e5a2-4832-ba33-ad33c92c5e4a">

Reply:
as discussed, we should use the `OpenAITool` class and just massage the data as necessary inline to convert

Reply:
actually for now let's just ignore this, we can fix this later when we work to move this out of beta into production